netdata/netdata
# Delay node update information (parent / child setup)

##### Summary
Currently when a child connects, there is a message that is sent to the cloud to indicate the child is live. Similarly on child disconnection, a message indicating the child has disconnected is sent. 

This PR will delay a bit to make sure the child will not immediately disconnect and avoid sending too many messages to the cloud. 

##### Test Plan
- Setup a parent (claimed to the cloud) with at least one child (activate aclk conversation logs)
   `conversation log = yes` in the `[cloud]` section
- Start the parent and then the child / children
   - Stop and restart the child -- notice disconnect and reconnect messages sent to the cloud
   - Look for `OUTGOING Msg:"UpdateNodeInstanceConnection"` messages in the `aclk.log` 
- Apply the PR and repeat
  - Notice less messages sent to the cloud as you stop and restart a child
